soccer-records-model/src/test/java/cz/muni/fi/pa165/team/TeamPlayerFacadeTest.java

```diff
@@ -0,0 +1,85 @@
+package cz.muni.fi.pa165.team;
+
+import cz.muni.fi.pa165.config.ApplicationConfig;
+import cz.muni.fi.pa165.team.exceptions.TeamPlayerNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTransactionalTestNGSpringContextTests;
+import org.testng.annotations.Test;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.UUID;
+
+import static org.testng.Assert.*;
+
+/**
+ * @author Libor Mühlpachr <libor.muhl@seznam.cz>
+ */
+public class TeamPlayerFacadeTest extends AbstractTransactionalTestNGSpringContextTests
+{
+
+    @Autowired
+    public TeamPlayerRepository teamPlayerRepository;
```

Comment:
už jsem komentoval Denisovi, zkus pls tu fasadu v tomhle testu nepoužít, koukni jak jsem to měnil u `UserFacadeTest`